common/test/views/support/cleaner/AtomCleanerTest.scala

```diff
@@ -32,7 +32,7 @@ class AtomCleanerTest extends FlatSpec with Matchers with FakeRequests {
 
 
  private def clean(document: Document, atom:Option[Atoms], amp: Boolean): Document = {
-    val cleaner = AtomsCleaner(youTubeAtom, shouldFence = false, amp = amp)(TestRequest())
+    val cleaner = AtomsCleaner(youTubeAtom, amp = amp)(TestRequest())
```

Comment:
No , because `shouldFence` is only used by interactive atoms, it should not really be an `AtomsCleaner` argument I think it could be pushed down into the interactive fragment.